com.unity.render-pipelines.universal/Shaders/PostProcessing/Bloom.shader

```diff
@@ -3,6 +3,7 @@ Shader "Hidden/Universal Render Pipeline/Bloom"
     HLSLINCLUDE
         #pragma exclude_renderers gles
         #pragma multi_compile_local _ _USE_RGBM
+        #pragma multi_compile _ _USE_FAST_SRGB_LINEAR_CONVERSION
```

Comment:
if this variant is only used in fragment you can add `_fragment` to multi_compile